packages/front-end/components/Features/ExperimentRefSummary.tsx

```diff
@@ -0,0 +1,328 @@
+          <strong>INCLUDE</strong>
+        </div>
+        <div className="col-auto">
+          <span className="mr-1 border px-2 py-1 bg-light rounded">
+            {percentFormatter.format(effectiveCoverage)}
+          </span>{" "}
+          of users in the experiment
+          {hasNamespace && (
+            <>
+              <span> (</span>
+              <span className="border px-2 py-1 bg-light rounded">
+                {percentFormatter.format(namespaceRange)}
+              </span>{" "}
+              of the namespace and{" "}
+              <span className="border px-2 py-1 bg-light rounded">
+                {percentFormatter.format(phase.coverage)}
+              </span>
+              <span> exposure)</span>
+            </>
+          )}
+        </div>
+      </div>
+      <strong>SERVE</strong>
+
+      <table className="table mt-1 mb-3 bg-light gbtable">
+        <tbody>
+          {experiment.variations.map((r, j) => (
+            <tr key={j}>
+              <td
+                className="text-muted position-relative"
+                style={{ fontSize: "0.9em", width: 25 }}
+              >
+                <div
+                  style={{
+                    width: "6px",
+                    position: "absolute",
+                    top: 0,
+                    bottom: 0,
+                    left: 0,
+                    backgroundColor: getVariationColor(j),
+                  }}
+                />
+                {j}.
+              </td>
+              <td>
+                <ValueDisplay
+                  value={rule.variations[j]?.value || ""}
+                  type={type}
+                />
+              </td>
+              <td>{r?.name}</td>
+              <td>
+                <div className="d-flex">
+                  <div
+                    style={{
+                      width: "4em",
+                      maxWidth: "4em",
+                      margin: "0 0 0 auto",
+                    }}
+                  >
+                    {percentFormatter.format(phase.variationWeights[j] || 0)}
+                  </div>
+                </div>
+              </td>
+            </tr>
+          ))}
+          <tr>
+            <td colSpan={4}>
+              <ExperimentSplitVisual
+                values={experiment.variations.map((v, i) => ({
+                  value: rule.variations[i]?.value || "",
+                  weight: phase.variationWeights[i] || 0,
+                  name: v.name,
+                }))}
+                coverage={effectiveCoverage}
+                label="Traffic split"
+                unallocated="Not included (skips this rule)"
+                type={type}
+                showValues={false}
+                stackLeft={true}
+                showPercentages={true}
+              />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <div className="row align-items-center">
+        <div className="col-auto">
+          <strong>TRACK</strong>
```

Comment:
I'm not sure how much of our app is accessible to screen readers but a best practice for formatting all uppercase is to do it with CSS instead of in the HTML in the event that the screen reader misreads it as an acronym. We can use the `text-uppercase` class. [Here's an example](https://github.com/growthbook/growthbook/blob/main/packages/front-end/components/EventWebHooks/EventWebHookList/EventWebHookList.tsx#L43).